app/models/ability.rb

```diff
@@ -50,8 +50,10 @@ def not_signed_in
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
     can :show, User
     unless CONFIG['authentication']['ichain']['enabled']
-      can [:show, :create], Registration do |registration|
-        registration.new_record?
+      can :show, Registration
+
```

Comment:
I don't quite recall myself why we allowed show method for new records of Registration and Event. You could open an issue about it, so that we can investigate it in more depth.
Let's keep it as it was till we do.